app/views/layouts/_lte_admin_sidebar.html.erb

```diff
@@ -116,6 +116,15 @@
       <p>FAQ</p>
     <% end %>
   </li>
+  
+  <% if Rails.env.production? %>
+    <li class="nav-item">
+      <%= link_to("/coverage", class: "nav-link #{"active" if current_page?(admin_questions_path)}") do %>
```

Comment:
This still isn't right?

Reply:
You are right, I got rid of the one above and didn't notice this one.